src/app/issure/advance-editor/advance-editor-config.ts

```diff
@@ -150,15 +150,6 @@ var editTextfieldCompValidation = [
         "key": "validateOn",
         "ignore": true
     },
```

Comment:
@suvarnakale is min and max length handled for number too?

Reply:
@Pratikshakhandagale Please check the ticket, I have described how min and max validation will be handled for number.